WordPress/src/main/AndroidManifest.xml

```diff
@@ -501,7 +501,7 @@
         <activity
             android:name=".ui.main.SitePickerActivity"
             android:label="@string/site_picker_title"
-            android:theme="@style/CalypsoTheme" />
+            android:theme="@style/WordPress.NoActionBar" />
```

Comment:
The plan is to phase out Calypso related styles and terminology in favor of WordPress.